src/shogun/CMakeLists.txt

```diff
@@ -461,6 +461,7 @@ IF (CTAGS_FOUND)
         # functions, classes, macroses, enumerations, enumerators, typedefs
         --c++-kinds=fcdgetp
         --fields=+im
+        -h "h.hpp"
```

Comment:
ah of course!!

Reply:
No that didn't help. But in the next patch I added something that helped ;)

Now we don't have nothing from .cpp in tags file.

Reply:
lol

well ... anyways :dancer: